Fixes following gcc7 issue. Need to be evaluated with upstream.

  CXX      clients/memflush.o
clients/memflush.cc: In function 'int main(int, char**)':
clients/memflush.cc:42:22: error: ISO C++ forbids comparison between pointer and integer [-fpermissive]
   if (opt_servers == false)
                      ^~~~~
clients/memflush.cc:51:24: error: ISO C++ forbids comparison between pointer and integer [-fpermissive]
     if (opt_servers == false)
                        ^~~~~

--- libmemcached-1.0.18/clients/memflush.cc
+++ libmemcached-1.0.18/clients/memflush.cc
@@ -39,7 +39,7 @@
 {
   options_parse(argc, argv);
 
-  if (opt_servers == false)
+  if (opt_servers == NULL)
   {
     char *temp;
 
@@ -48,7 +48,7 @@
       opt_servers= strdup(temp);
     }
 
-    if (opt_servers == false)
+    if (opt_servers == NULL)
     {
       std::cerr << "No Servers provided" << std::endl;
       exit(EXIT_FAILURE);
